awx/main/models/events.py

```diff
@@ -579,6 +582,13 @@ def _update_host_summary_from_stats(self, hostnames):
                     h.last_job_host_summary_id = host_mapping[h.id]
                     updated_hosts.add(h)
 
+                # Populate unique host retention table if enabled
+                if settings.UNIQUE_HOST_RETENTION_ENABLED:
+                    # If host already exists, update it, else create it
+                    UniqueHost.objects.update_or_create(
+                        host_name=h.name, defaults={'created': now(), 'modified': now(), 'job_host_summary_id': h.last_job_host_summary_id, 'host_id': h.id}
```

Comment:
In this case isn't `created` and `modified` automatically applied from the base class? Is there a reason we need to call  `now()` explicitly at this level?

Reply:
I *think* the explicit act of calling save is all you need to do?

https://github.com/ansible/awx/blob/ba324c73ceefa22d516d872527649d090d3bc0cc/awx/main/models/base.py#L166

or is that not called explicitly when it's happening in bulk?

Reply:
Apparently so, removed the created/modified.  Thanks